### ISSUE TITLE
combineScripts: true causing empty file name generation

### ISSUE THREAD

**Issue by geoffreymcgill** [bridgedotnet/Bridge#1300]
### Expected

```
demo.js
demo.min.js
```

### Actual

```
.js  <-- no name
.min.js
```

### Steps To Reproduce

**bridge.json**

```javascript
{
  "output": "Bridge/output",
  "combineScripts": true
}
```

**Comment by geoffreymcgill** [comment]
Related to #1284

**Comment by LeonidVeriga** [comment]
The issue fixed in #1284.

Changed Compiler test 02 to run it using Task Builder.
